info/available-rules.md

```diff
@@ -1,129 +1,130 @@
-|    6  | 6.2.2    |  EXTENSION_FUNCTION_SAME_SIGNATURE       | Checks if extension function has the same signature as another extension function and their classes are related                                                                                                                     |  no  |  no  |  + |
-|    6  | 6.4.1    |  AVOID_USING_UTILITY_CLASS               | Checks if there is class/object that can be replace with extension function                                                                                                                                                         | no  | no | - |
-|    6  | 6.4.2    |  OBJECT_IS_PREFERRED                     | Checks: if class is stateless it is preferred to use `object`                                                                                                                                                                       | yes | no | + |
-|    6  | 6.2.3    | EXTENSION_FUNCTION_WITH_CLASS            | Check: if file contains class, then it can not have extension functions for the same class                                                                                                                                          | no | no | - |
-|    6  | 6.5.1    |  RUN_IN_SCRIPT                           | Checks : if kts script contains other functions except run code                                                                                                                                                                     | yes | no | - |
-| 6    | 6.2.4    | USE_LAST_INDEX                            | Check: should change property length - 1 to property lastIndex                                                                                                                                                                   | no  | no                                                                                                                                                                                                                         | -                                                                                                                                                                                                                             |
\ No newline at end of file
+| Chap  | Standard | Rule name                                 | Description                                                                                                                                                                                                                         | Fix | Config                                                                                                                                                                                                                     | FixMe |
```

Comment:
btw - why do we see changes here? Line ending again?

Reply:
looks like, also IDEA adds additional `-` to header. 
Will look at it